analyze_ipa.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const (
+	contentsFolder = "contents"
+	payloadFolder  = contentsFolder + "/Payload"
+	plistFilePath  = "/Info.plist"
+)
+
+func main() {
+	args := os.Args
+	if len(args) < 2 {
+		log.Fatal("No .ipa file specified. Usage: `go run bitrise-trial path/to/file.ipa`")
+	}
+	filePath := args[1]
+
+	unzip(filePath, contentsFolder)
+	plistPath := plistPath(payloadFolder)
+
+	result := parsePlist(plistPath)
+	printResults(result)
+
+	os.RemoveAll(contentsFolder)
```

Comment:
if the program exists before this line, the temporary `contentsFolder` will not be cleaned up, you could use defer this cleanup right after the `unzip` to make sure it always gets cleaned